<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDialog1</class>
 <widget class="QDialog" name="UserDialog1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_9">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_13" stretch="159,268,93">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QLabel" name="label_chooseVaccine">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>707</width>
           <height>240</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>651</width>
           <height>221</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="pixmap">
          <pixmap>:/img/vaccinePic1.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="5,3">
       <property name="spacing">
        <number>30</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9" stretch="1,1">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,9">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_vaccine_version_2">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <italic>false</italic>
               <bold>true</bold>
               <underline>true</underline>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;
color: black;
font-weight: bold;</string>
             </property>
             <property name="text">
              <string>Enter your details :</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,1">
             <property name="spacing">
              <number>20</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,8">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="rightMargin">
                <number>5</number>
               </property>
               <item>
                <widget class="QLabel" name="label_aadhaar">
                 <property name="styleSheet">
                  <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Aadhaar No.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_aadhaar">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,8">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="rightMargin">
                <number>5</number>
               </property>
               <item>
                <widget class="QLabel" name="label_fName">
                 <property name="styleSheet">
                  <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="text">
                  <string>First Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_fName">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,8">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="rightMargin">
                <number>5</number>
               </property>
               <item>
                <widget class="QLabel" name="label_lName">
                 <property name="styleSheet">
                  <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Last Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_lName">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,9">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="label_vaccine_version">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <italic>false</italic>
               <bold>true</bold>
               <underline>true</underline>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;
color: black;
font-weight: bold;</string>
             </property>
             <property name="text">
              <string>Vaccine :</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1">
             <property name="spacing">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="radioButton_Covishield">
               <property name="styleSheet">
                <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Covishield</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_Covaxin">
               <property name="styleSheet">
                <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Covaxin</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_SputnikV">
               <property name="styleSheet">
                <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Sputnik - V</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10" stretch="1,1">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,9">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="label_age">
             <property name="styleSheet">
              <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;
color: black;
font-weight: bold;</string>
             </property>
             <property name="text">
              <string>Age :</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1">
             <property name="spacing">
              <number>20</number>
             </property>
             <item>
              <widget class="QRadioButton" name="radioButton_under18">
               <property name="styleSheet">
                <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Under  18</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_over18">
               <property name="styleSheet">
                <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Between 18 - 45</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_over45">
               <property name="styleSheet">
                <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Over 45</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8" stretch="1,9">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="label_dose">
             <property name="styleSheet">
              <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;
color: black;
font-weight: bold;</string>
             </property>
             <property name="text">
              <string>Dose :</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1">
             <property name="spacing">
              <number>20</number>
             </property>
             <item>
              <widget class="QRadioButton" name="radioButton_firstDose">
               <property name="styleSheet">
                <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>First Dose</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_secondDose">
               <property name="styleSheet">
                <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>Second Dose</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_hours">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>294</width>
                 <height>41</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="insertPolicy">
                <enum>QComboBox::InsertAtBottom</enum>
               </property>
               <item>
                <property name="text">
                 <string>09:00 - 11:00 a.m.</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>11:00 - 01:00 p.m.</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>01:00 - 03:00 p.m.</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>03:00 - 05:00 p.m.</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12" stretch="1,2">
       <property name="spacing">
        <number>30</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="rightMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="label_date">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: black;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
             </property>
             <property name="text">
              <string> Vaccination date:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <italic>false</italic>
               <bold>false</bold>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="whatsThis">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="keyboardTracking">
              <bool>true</bool>
             </property>
             <property name="maximumDateTime">
              <datetime>
               <hour>23</hour>
               <minute>59</minute>
               <second>59</second>
               <year>2022</year>
               <month>12</month>
               <day>31</day>
              </datetime>
             </property>
             <property name="maximumDate">
              <date>
               <year>2022</year>
               <month>12</month>
               <day>31</day>
              </date>
             </property>
             <property name="minimumDate">
              <date>
               <year>2022</year>
               <month>1</month>
               <day>11</day>
              </date>
             </property>
             <property name="minimumTime">
              <time>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
              </time>
             </property>
             <property name="currentSection">
              <enum>QDateTimeEdit::DaySection</enum>
             </property>
             <property name="displayFormat">
              <string>dd-MMM-yyyy</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
             <property name="timeSpec">
              <enum>Qt::LocalTime</enum>
             </property>
             <property name="date">
              <date>
               <year>2022</year>
               <month>1</month>
               <day>11</day>
              </date>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="8,5">
         <property name="spacing">
          <number>7</number>
         </property>
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_userP1">
           <property name="styleSheet">
            <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1,2">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_userLoadData">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Load Data</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_userSubmit">
             <property name="styleSheet">
              <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Submit</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
